openshift/buildConfig-template.yaml

```
@@ -263,13 +263,87 @@ objects:
     metadata:
       name: "solver-rhel-8.0-py36"
       annotations:
-        thoth-station.ninja/template-version: 0.9.1
+        thoth-station.ninja/template-version: 1.4.0
       labels:
         app: "thoth"
         component: "solver"
         app.kubernetes.io/name: "thoth"
         app.kubernetes.io/component: "solver"
         app.thoth-station.ninja/solver: "rhel-8.0-py36"
+    spec:
+      output:
+        to:
+          kind: "ImageStreamTag"
+          name: "solver-rhel-8.0-py36:${IMAGE_STREAM_TAG}"
+      source:
+        dockerfile: |
+          FROM registry.access.redhat.com/ubi8/python-36:1-58
+
+          ENV LANG=en_US.UTF-8 \
+              THOTH_SOLVER_TMP_DIR='/tmp/thoth-solver-install'
+
+          USER 0
+          RUN useradd -m solver -s /sbin/nologin -c "Thoth Solver User"
+
+          USER ${USERID}
+          RUN mkdir -p ${THOTH_SOLVER_TMP_DIR}
+
+          # Install thoth-solver itself
+          COPY ./ ${THOTH_SOLVER_TMP_DIR}
+          RUN cd ${THOTH_SOLVER_TMP_DIR} &&\
+              pip3 install --upgrade setuptools pip &&\
```

@goern can you change this to :
```suggestion
              pip3 install --upgrade setuptools pip &&\
    use
             `/usr/bin/python3.6 -m pip install --upgrade setuptools pip`

```